dxr/templates/layout.html

```diff
@@ -49,20 +49,25 @@
                           </button>
                         </section>
                         <div class="sf-select-options sf-modal" aria-expanded="false">
-                          <ul class="selector-options" tabindex="-1">
+                          <table class="selector-options" tabindex="-1">
                             {% for filter in filters %}
-                              <li>
-                                <a href="javascript:void(0)" data-value="{{ filter.name }}:">
-                                  <span class="selector-option-label">
-                                    {{ filter.name }}
-                                  </span>
-                                  <span class="selector-option-description">
-                                    {{ filter.description }}
-                                  </span>
-                                </a>
-                              </li>
+                              <tr data-value="{{ filter.name }}:">
+                                <td class="selector-option-label">
+                                  {{ filter.name }}
```

Comment:
For that matter, I thought filters were now being listed in alphabetical order. They don't seem to be. Hmm.